tests/scanners/test.javascript.js

```diff
@@ -227,6 +228,45 @@ describe('JavaScript Scanner', function() {
     assert(ok);
   });
 
+  it('should pass addon metadata to rules', () => {
+    var fakeRules = { metadata_not_passed: () => {} };
+
+    sinon.stub(fakeRules, 'metadata_not_passed', (context) => {
+      return {
+        Identifier: (node) => {
+          var metadata = context.settings.addonMetadata;
+
+          if (!metadata || typeof metadata.guid === 'undefined') {
```

Comment:
Duh, I don't know why but I didn't spot this was a test  - ignore me!

![dont-panic-and-carry-on1](https://cloud.githubusercontent.com/assets/1514/11397523/16079aba-9372-11e5-8fba-e4898fa17ca3.png)